modules/dmrpp_module/build_dmrpp_util.cc

```diff
@@ -392,20 +397,267 @@ get_value_as_string(hid_t h5_type_id, vector<char> &value)
             throw UnsupportedTypeException(msg);
         }
         case H5T_COMPOUND: {
-            string msg(prolog + "UnsupportedTypeException: Your data granule contains an H5T_COMPOUND as user-defined fillValue type. "
-                   "This is not yet supported by the dmr++ creation machinery. ");
             string str_fv(value.begin(),value.end());
-            throw UnsupportedTypeException(msg);
+            break;
```

Comment:
Here I think the value of str_fv should be returned. I think no value is returned for this case.

Reply:
The compound datatype fill value is calculated elsewhere.  Here we need to use break only.